core/src/main/java/hudson/Functions.java

```diff
@@ -2289,4 +2290,59 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    public static Icon tryGetIcon(String iconGuess) {
```

Comment:
I don't think we need this in the public API?
```suggestion
    @Restricted(NoExternalUse.class)
    public static Icon tryGetIcon(String iconGuess) {
```